client/src/screens/events/Detail.js

```diff
@@ -1,22 +1,23 @@
 /* global navigator */
 import PropTypes from 'prop-types';
 import React, { Component } from 'react';
-import { Text, View, Dimensions, StyleSheet } from 'react-native';
+import { Text, View, Dimensions, StyleSheet, PermissionsAndroid, Platform } from 'react-native';
 import MapView, { Marker } from 'react-native-maps';
 import { graphql, compose } from 'react-apollo';
 import { connect } from 'react-redux';
 import moment from 'moment';
-import _ from 'lodash';
+import { _, isEqual } from 'lodash';
```

Comment:
why did you do this? One of the hallmark features of `lodash` and `underscore` is that you call all of its bits using `_.*()`.

Reply:
?

Reply:
whoops. missed that one